mkdocs.yml

```diff
@@ -81,7 +81,7 @@ nav:
         - Init: reference/usage/init.md
         - Run: reference/usage/run.md
         - Report: reference/usage/report.md
-        - Notify: reference/usage/notification.md
+        - Notifications: reference/usage/notification.md
```

Comment:
Interestingly, we chopse Notify for its action driven aspect. They are all verbs, not nouns.

Reply:
I'd be inclined to agree, if it were an actual command you can run, but it isn't

Chaos Toolkit can do notifications, but you can't do `chaos notify`

Reply:
Mh, that makes sense indeed. Actually, that whole notifications mechanic is obsleted by controls anyway. But controls we haven't shipped yet. Okay make the change for now.